ArduPlane/quadplane.cpp

```diff
@@ -2252,7 +2252,10 @@ void QuadPlane::vtol_position_controller(void)
 void QuadPlane::setup_target_position(void)
 {
     const Location &loc = plane.next_WP_loc;
-    Location origin = inertial_nav.get_origin();
+    Location origin;
+    if (!ahrs.get_origin(origin)) {
+        origin.zero();
```

Comment:
OK, no change in behaviour here.  OTOH, I'm really not sure we shouldn't use whatever garbage ahrs returns us rather than the wrong (0,0) value.

Reply:
I don't disagree but was doing the no change approach :)